src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -134,4 +136,25 @@ public File getPluginDirectoryPath() {
     public void setPluginDirectoryPath(File pluginDirectoryPath) {
         this.pluginDirectoryPath = pluginDirectoryPath;
     }
+
+    /**
+     * This method reads multiple values set manually by editing the project configuration
+     * and then setting it in the GUI so that the person who is having the privileges to the UI is aware of the 
+     * parameters set manually.
+     * Ex:
+     * [eiffel-integration]
+     *    filter = branch1, branch2
+     *    filter = branch3
+     * @param key  the property which can contain multiple values, for example: Filter, FlowContext, etc.
+     * @param pluginConfig the configuration for the project read
+     * @return String  returns the value for the property read from the plugin configuration.
+     */
+    private String getMultiValueParameters(final String key, final PluginConfig pluginConfig){
+        String keyValue = "";
+        if (pluginConfig.getStringList(key)!=null && pluginConfig.getStringList(key).length > 0) {
```

Comment:
Please move the filter string list to a local variable so you only need to use .getStringList once. Once this is done you may use StringUtils.isEmpty(filterList.toString()) StringUtils checks both fotr null and empty.

Reply:
fixed in https://github.com/eiffel-community/eiffel-gerrit-plugin/pull/35/commits/22af4864a1073d3c6e87e840dd6338d9ea7757b1